tests/ui/async-await/pin-ergonomics/pinned-local.stderr

```diff
@@ -0,0 +1,35 @@
+error[E0384]: cannot assign twice to immutable variable `y`
+  --> $DIR/pinned-local.rs:12:5
+   |
+LL |     let pin const y = Foo;
+   |         ----------- first assignment to `y`
+LL |     x = Foo; // FIXME: this should be an error
+LL |     y = Foo;
+   |     ^^^^^^^ cannot assign twice to immutable variable
+   |
+help: consider making this binding mutable
+   |
+LL |     let mut pin const y = Foo;
+   |         +++
```

Comment:
This is wrong.

Reply:
I removed this suggestion as it is out of the scope of this PR.